### PR TITLE
i18n(ko-KR): update `migrate-to-astro/`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)
- update `migrate-to-astro/`
- #9240
- #10159 
- #10160 
- #10161

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10166, #10205, #10228 are merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
